src/commands/deploy/deploy.ts

```diff
@@ -0,0 +1,52 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+import { VerifyProvidersStep } from "@microsoft/vscode-azext-azureutils";
+import { AzureWizard, AzureWizardExecuteStep, AzureWizardPromptStep, ITreeItemPickerContext, createSubscriptionContext } from "@microsoft/vscode-azext-utils";
+import { webProvider } from "../../constants";
+import { ContainerAppItem } from "../../tree/ContainerAppItem";
+import { localize } from "../../utils/localize";
+import { pickContainerApp } from "../../utils/pickContainerApp";
+import { ContainerAppOverwriteConfirmStep } from "./ContainerAppOverwriteConfirmStep";
+import { ContainerAppUpdateStep } from "./ContainerAppUpdateStep";
+import { IDeployBaseContext } from "./IDeployBaseContext";
+import { ImageSourceListStep } from "./ImageSourceListStep";
+
+export async function deploy(context: ITreeItemPickerContext & Partial<IDeployBaseContext>, node?: ContainerAppItem): Promise<void> {
+    if (!node) {
+        context.suppressCreatePick = true;
+        node = await pickContainerApp(context);
+    }
+
+    const { subscription, containerApp } = node;
+
+    const wizardContext: IDeployBaseContext = {
+        ...context,
+        ...createSubscriptionContext(subscription),
+        subscription,
+        targetContainer: containerApp
+    };
+
+    const title: string = localize('deploy', 'Deploying to "{0}"', containerApp.name);
```

Comment:
It looks like this variable is only used in one place below on line 44. Can we just create this value inline on line 44 instead of creating a constant for it here?